### ISSUE TITLE
Bad filename results in cryptic error

### ISSUE THREAD

**Issue by drozzy** [datapane/datapane#103]
If you call `report.save` with a super long (or invalid filename), you get a really weird error. At first I thought something was wrong with my plots, but nope it was just a filename issue.

```
Error validating report document:

.....

validate_report_doc
    rng_validator.assertValid(xml_doc)
  File "src/lxml/etree.pyx", line 3623, in lxml.etree._Validator.assertValid
lxml.etree.DocumentInvalid: Error validating datatype string

```

**Comment by mands** [comment]
Thanks for raising, i think I know why this is occurring - we'll look into and try publish a fix in the next week or so.

**Comment by mands** [comment]
fixed in b9b41ea2abceb8971c26948056f089e2d0f7d104
